custom-connectors/Elfsquad Data/apiProperties.json

```diff
@@ -0,0 +1,34 @@
+{
+    "properties": {
+      "capabilities": [],
+      "connectionParameters": {
+        "token": {
+          "oAuthSettings": {
+            "clientId": "68CC9851-4AA0-459A-98FB-07742B384987",
+            "customParameters": {
+              "authorizationUrl": {
+                "value": "https://login.elfsquad.io/connect/authorize"
+              },
+              "refreshUrl": {
+                "value": "https://login.elfsquad.io/connect/token"
+              },
+              "tokenUrl": {
+                "value": "https://login.elfsquad.io/connect/token"
+              }
+            },
+            "identityProvider": "oauth2",
+            "properties": {
+              "IsFirstParty": "False"
+            },
+            "redirectMode": "Global",
+            "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+            "scopes": [
+              "openid profile offline_access Elfskot.Api"
+            ]
+          },
+          "type": "oauthSetting"
+        }
+      },
+      "iconBrandColor": "#313130"
```

Comment:
@johannesheesterman Please add `publisher` and `stackOwner` Properties.

Documentation - [Adding Metadata](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#adding-metadata)

Reply:
@sriyen-msft I have added the requested properties.